libminifi/include/sitetosite/RawSocketProtocol.h

```diff
@@ -178,9 +178,9 @@ class RawSiteToSiteClient : public sitetosite::SiteToSiteClient {
   virtual bool handShake();
 
  private:
-
+  /*
   // Mutex for protection
```

Comment:
Might as well remove it if it was not referenced anywhere.

Reply:
Removed, thanks!